docs/config.json

```
@@ -35,7 +35,11 @@
             {
               "title": "Upgrading Reference",
               "slug": "/upgrading/upgrading-reference/"
-	    }
+	        },
+            {
+              "title": "Client Tools Automatic Updates",
```

```suggestion
              "title": "Client Tool Automatic Updates",
```
E.g., for the reason why we say "Database Management System" instead of "Databases Management System", or "toothbrush" instead of "teethbrush".